includes/settings/class-amp-customizer-design-settings.php

```diff
@@ -62,37 +127,97 @@ public static function register_customizer_ui( $wp_customize ) {
 		// Header background color control.
 		$wp_customize->add_control(
 			new WP_Customize_Color_Control( $wp_customize, 'amp_header_background_color', array(
-				'settings'   => 'amp_customizer[header_background_color]',
+				'settings' => 'amp_customizer[header_background_color]',
 				'label'    => __( 'Header Background & Link Color', 'amp' ),
 				'section'  => 'amp_design',
 				'priority' => 20,
 			) )
 		);
 
-		// Background color scheme
+		// Background color scheme.
 		$wp_customize->add_control( 'amp_color_scheme', array(
 			'settings'   => 'amp_customizer[color_scheme]',
 			'label'      => __( 'Color Scheme', 'amp' ),
 			'section'    => 'amp_design',
 			'type'       => 'radio',
 			'priority'   => 30,
 			'choices'    => self::get_color_scheme_names(),
-		));
+		) );
+
+		// Header.
+		$wp_customize->selective_refresh->add_partial( 'amp-wp-header', array(
+			'selector'         => '.amp-wp-header',
+			'settings'         => array( 'blogname' ), // @todo Site Icon.
+			'render_callback'  => array( __CLASS__, 'render_header_bar' ),
+			'fallback_refresh' => false,
+		) );
+
+		// Header.
+		$wp_customize->selective_refresh->add_partial( 'amp-wp-footer', array(
+			'selector'            => '.amp-wp-footer',
+			'settings'            => array( 'blogname' ),
+			'render_callback'     => array( __CLASS__, 'render_footer' ),
+			'fallback_refresh'    => false,
+			'container_inclusive' => true,
+		) );
+	}
+
+	/**
+	 * Render header bar template.
+	 */
+	public static function render_header_bar() {
+		if ( is_singular() ) {
+			amp_load_classes();
```

Comment:
Make a note to remove all `amp_load_classes()` in #828.

Reply:
Done